tests/src/FunctionalJavascript/JavascriptBehavioursTest.php

```diff
@@ -31,6 +34,15 @@ class JavascriptBehavioursTest extends WebDriverTestBase {
    */
   protected $defaultTheme = 'stark';
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function tearDown(): void {
+    $this->failOnJavascriptErrors();
```

Comment:
Looks like failOnJavascriptErrors in this place doesn't catch js errors.

Reply:
Exposing of js error in this place doesn't work if we had some failure before.
Also, it will not throw error if previously did `$this->failOnJavascriptErrors()`.